cpp/include/cudf/stream_compaction.hpp

```diff
@@ -164,15 +164,15 @@ std::unique_ptr<experimental::table>
  * If `ignore_nulls` is true and `nan_as_null` is false, only `null` is
  * ignored, `NaN` is considered in unique count.
  *
- * @param[in] input         The column_view whose unique elements will be counted.
- * @param[in] ignore_nulls  flag to ignore `null` in unique count if true
- * @param[in] nan_as_null   flag to consider `NaN==null` if true.
+ * @param[in] input The column_view whose unique elements will be counted.
+ * @param[in] _include_nulls flag to include or ignore `null` while counting
+ * @param[in] nan_as_null flag to consider `NaN==null` if true.
  * @param[in] mr Optional, The resource to use for all allocations
  *
  * @return number of unique elements
  */
 cudf::size_type unique_count(column_view const& input,
-                             bool const& ignore_nulls,
+                             include_nulls const& _include_nulls,
```

Comment:
```suggestion
                             include_nulls const _include_nulls,
```
Maybe `_include_nulls` and `nan_as_null` could be passed by value instead of by reference?